PROPOSALS.md

```diff
@@ -54,6 +62,15 @@ The Core Libraries Committee's focus is around management,
 not implementation: you should do the work for your own proposals
 (or others' if you like them!) to show us that you think it's worth doing.
 
+To be more precise, as the proposal author, you're expected to do the following
```

Comment:
The proposal process is described in more details at https://github.com/haskell/core-libraries-committee/blob/main/PROPOSALS.md. Should we merge these documents?..

Reply:
@Bodigrim Sorry, I didn't quite get your suggestion. Could you elaborate a bit more, please?

I already made my changes in the existing `PROPOSALS.md` document instead of creating a new one. Did you probably mean that it makes sense to merge `README.md` and `PROPOSALS.md`?